### DISCUSSION TITLE
T&C's

### DISCUSSION THREAD

**Discussion by PaulEasterMonash** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#65] [General]
Hi,

Found the T&C on overview tab on website, it didn't load beforehand so I missed it.

Cheers,

Paul

**Comment by anprieto** [comment]
Yes, it's there:
![image](https://user-images.githubusercontent.com/4283049/118934598-cfcd7880-b94a-11eb-90b3-7defb3d407d3.png)
